### DISCUSSION TITLE
How to Do SDXL Training For FREE with Kohya LoRA - Kaggle Notebook - NO GPU Required - Pwns Google Colab - 53 Chapters - Manually Fixed Subtitles

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [bmaltais/kohya_ss#1474] [Show and tell]
I also made a pull request @bmaltais I hope you can approve it. Keep up the amazing work.

[**How To Do SDXL LoRA Training On RunPod With Kohya SS GUI Trainer & Use LoRAs With Automatic1111 UI**](https://youtu.be/JF2P7BIUpIU?feature=shared)

[![image](https://cdn-uploads.huggingface.co/production/uploads/6345bd89fe134dfd7a0dba40/n82kc7ND2rDmhRmRexLrb.png)](https://youtu.be/JF2P7BIUpIU?feature=shared)

**Comment by Huzaif-Ahmed** [comment]
thank you for such amazing content ,i just wanted to ask that i havr given a training image dataset of 5 images and it shows me 4 hrs .but i wanted to run in p100 gpu provided by kaggke

so i made changes like
!accelerate launch --num_gpus=1 "./sdxl_train_network.py" \
 --pretrained_model_name_or_path="stabilityai/stable-diffusion-xl-base-1.0" \
 --train_data_dir="/kaggle/working/training-data/img" \
 --reg_data_dir="/kaggle/working/training-data/reg" \
 --output_dir="/kaggle/working/my-dreambooth-lora" \
 --logging_dir="/kaggle/working/training-data/log" \
 --resolution="1024,1024" \
 --network_alpha="1" \
 --network_dim=32 \
 --save_model_as=safetensors \
 --network_module=networks.lora \
 --text_encoder_lr=0.0004 \
 --unet_lr=0.0004 \
 --output_name="sdxl-dreambooth-lora" \
 --lr_scheduler_num_cycles="8" \
 --no_half_vae \
 --learning_rate="0.0004" \
 --lr_scheduler="constant" \
 --train_batch_size="1" \
 --max_train_steps=$(($(ls /kaggle/input/training-images | wc -l) * 400)) \
 --save_every_n_epochs="1" \
 --mixed_precision="amp" \
 --save_precision="amp" \
 --optimizer_type="Adafactor" \
 --optimizer_args scale_parameter=False relative_step=False warmup_init=False \
 --max_data_loader_n_workers="0" \
 --gradient_checkpointing \
 --xformers \
 --bucket_no_upscale \
 --noise_offset=0.0 \
 --lowram \
 --mem_eff_attn
 
 still itd not wokring can you please help me over here

**Comment by FurkanGozukara** [comment]
> thank you for such amazing content ,i just wanted to ask that i havr given a training image dataset of 5 images and it shows me 4 hrs .but i wanted to run in p100 gpu provided by kaggke
> 
> so i made changes like !accelerate launch --num_gpus=1 "./sdxl_train_network.py" --pretrained_model_name_or_path="stabilityai/stable-diffusion-xl-base-1.0" --train_data_dir="/kaggle/working/training-data/img" --reg_data_dir="/kaggle/working/training-data/reg" --output_dir="/kaggle/working/my-dreambooth-lora" --logging_dir="/kaggle/working/training-data/log" --resolution="1024,1024" --network_alpha="1" --network_dim=32 --save_model_as=safetensors --network_module=networks.lora --text_encoder_lr=0.0004 --unet_lr=0.0004 --output_name="sdxl-dreambooth-lora" --lr_scheduler_num_cycles="8" --no_half_vae --learning_rate="0.0004" --lr_scheduler="constant" --train_batch_size="1" --max_train_steps=$(($(ls /kaggle/input/training-images | wc -l) * 400)) --save_every_n_epochs="1" --mixed_precision="amp" --save_precision="amp" --optimizer_type="Adafactor" --optimizer_args scale_parameter=False relative_step=False warmup_init=False --max_data_loader_n_workers="0" --gradient_checkpointing --xformers --bucket_no_upscale --noise_offset=0.0 --lowram --mem_eff_attn
> 
> still itd not wokring can you please help me over here

you need to add full fp16 now
